sql-db/hibernate-reactive/src/main/resources/application.properties

```diff
@@ -0,0 +1,5 @@
+quarkus.hibernate-orm.database.charset=utf-8
+quarkus.hibernate-orm.database.generation=drop-and-create
+
+quarkus.hibernate-orm.provider=org.hibernate.reactive.provider.ReactivePersistenceProvider
+quarkus.hibernate-orm.class=io.quarkus.ts.reactive.Book, io.quarkus.ts.reactive.Author
```

Comment:
what are these properties doing? can you please attach here the link where they described?  
If it's for purpose, then please create an issue that following props are missing in https://quarkus.io/guides/all-config

Reply:
Looks like this property is not needed for Panache. Removed it.